src/Core/Domain/Product/ValueObject/Isbn.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Product\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductConstraintException;
+
+/**
+ * Holds product ISBN code value
+ */
+class Isbn
+{
+    /**
+     * Valid ISBN regex pattern
+     */
+    const VALID_PATTERN = '/^[0-9-]{0,32}$/';
```

Comment:
I wonder if those const should be in the ValueObject or in a dedicated interface/class for ISBN, or in a dedicated validation class that contains those constants 🤔

Because if we need them somewhere else it feels weird to have a relation with the ValueObject that is supposed to be used for command only

Reply:
yes you are right, this is some kind of global standard, idk how should we approach it ?
Even Symfony has such constraint for form validation, so it might introduce some strange behavior if for some reason it doesn't do the same :/

Reply:
yep we can merge this like this for now, but we need to think about it